### ISSUE TITLE
Test set

### ISSUE THREAD

**Issue by akashgokul** [xyutao/fscil#12]
For datasets like CUB, do you have a test.txt? If not, how did you gather your test dataset (is it all the test samples for each class in each session)?

**Comment by akashgokul** [comment]
Could you push a test_session_i.txt for each session?

**Comment by xyutao** [comment]
Hi @akashgokul , The test splits for CUB200 have been uploaded to the folder data/cub200.  The list of the test images for each session's classes is stored in the file "test_i.txt". You may notice that when evaluating the model at session t, please accumulate all the test images of the encountered sessions, from "test_1.txt" to "test_t.txt".

The original train-test split of CUB200 has been given by the official dataset files. You may download CUB200, unzip it, put it under the folder "data", and then run the split script "data/cub200_split.py" to reproduce the above test splits.

**Comment by akashgokul** [comment]
Okay thank you so much! 

I'm sorry to keep bothering but could you push the test_i.txt for the cifar100 and miniImageNet datasets? Sorry to keep asking,  but this makes reproducing/comparision using other libraries/methods much easier.

**Comment by xyutao** [comment]
Hi, the test_i.txt of cifar100 and miniImageNet have been generated and pushed. Please check out the updates.

**Comment by akashgokul** [comment]
Is there a reason that for CUB200, test_1.txt does not include the 100th class but session_1.txt does?
Is [this](https://github.com/xyutao/fscil/blob/master/data/index_list/cub200_split.py#L76) supposed to be range(start_class, end_class + 1)?

**Comment by xyutao** [comment]
Sorry, it's a bug. Now it's fixed. Thanks for pointing it out.

**Comment by akashgokul** [comment]
Thank you for the help!

Did you have this error when reporting your accuracies in the paper and README?

**Comment by xyutao** [comment]
No. The results in the paper and README are evaluated using the dataloader script, which is correct. The provided text files are generated alone from a new script cub200_split.py which was written several days ago for ease of use.
